src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsHostExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host.Config;
+using Microsoft.Azure.WebJobs.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace Microsoft.Extensions.Hosting
```

Comment:
So now this becomes the primary required namespace that needs to be added in a console app to use webjobs? Previously you added Microsoft.Azure.WebJobs, and all the UseXXX extension methods required were also in that namespace making all the required operations intellisense discoverable.

Reply:
Wouldn't say primary, but with the current design, that does become one of the required ones.